ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -66,6 +160,47 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
             repoSlug="repo"
           />
         );
+      //test initial state
+      assert.equal(
+        vocabularyComponentRendered.getInitialState().terms.length,
```

Comment:
You should be able to just assert `state.terms.length` here. `state` should have `getInitialState()`s values before any `setState` is called.